xml/obs/systemsmanagement:Agama:Devel.xml

```diff
@@ -4,7 +4,7 @@
     distri="opensuse"
     version="agama-9.0">
     <batch name="base" archs="x86_64 aarch64 ppc64le">
-        <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/" iso="1" media1="0"/>
+        <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/" iso="iso" extract_as_repo="1" media1="0"/>
```

Comment:
I have one question here, is there any difference  for s390x?

Reply:
REPO_999=agama-installer.s390x-9.0.0-openSUSE-Build17.9.iso \
 s390x has ln_iso_to_repo, so it has this setting.

Reply:
Thanks！